Assets/LeapMotion/Modules/InteractionEngine/Scripts/InteractionXRController.cs

```diff
@@ -99,9 +124,132 @@ public class InteractionXRController : InteractionController {
+           + "controller.isJoystickDetected and controller.isTracked are useful for this.")]
+    [SerializeField]
+    private List<GameObject> _enableObjectsOnlyWhenTracked;
+    /// <summary>
+    /// These objects will be made active only while the controller is tracked. For more
+    /// fine-tuned behavior, we recommend implementing your own logic.
+    /// 
+    /// controller.isJoystickDetected and controller.isTracked are useful for this.
+    /// </summary>
+    public List<GameObject> enableObjectsOnlyWhenTracked {
+      get {
+        if (_enableObjectsOnlyWhenTracked == null) {
+          _enableObjectsOnlyWhenTracked = new List<GameObject>();
+        }
+        return _enableObjectsOnlyWhenTracked;
+      }
+    }
+
+    #endregion // Inspector
+
+    #region Unity Events
+
+    protected override void Reset() {
+      base.Reset();
+
+      hoverEnabled = true;
+      contactEnabled = true;
+      graspingEnabled = true;
+
+      trackingProvider = _defaultTrackingProvider;
+      _hoverPoint = null;
+      primaryHoverPoints.Clear();
+      graspPoint = null;
+
+      maxGraspDistance = 0.06F;
+      graspTimingSlop = 0.1F;
+    }
+
+    protected virtual void OnValidate() {
+      _trackingProviderType = trackingProvider.GetType().ToString();
+    }
+
+    protected override void Start() {
+      base.Start();
+
+      trackingProvider.OnTrackingDataUpdate += refreshControllerTrackingData;
+    }
+
+    protected override void fixedUpdateController() {
+      fixedUpdatePollConnection();
+
+      if (isTracked) {
+        foreach (var gameObject in enableObjectsOnlyWhenTracked) {
+          gameObject.SetActive(true);
+        }
+      }
+      else {
+        foreach (var gameObject in enableObjectsOnlyWhenTracked) {
+          gameObject.SetActive(false);
+        }
+      }
+
+      refreshContactBoneTargets();
+    }
+
+    #endregion
+
+    #region Controller Connection Polling
+
+    private bool _isJoystickDetected = false;
+    /// <summary>
+    /// Whether the device joystick tokens matched an entry in Input.GetJoystickNames().
+    /// If pollConnection is set to true, this status is refreshed periodically based on
+    /// the pollConnectionInterval, but only while the joystick tokens have not been
+    /// detected from Input.GetJoystickNames(). Call RefreshControllerConnection() to
+    /// detect if the controller has been disconnected.
+    /// 
+    /// Joystick detection is skipped if deviceJoystickTokens is null or empty, causing
+    /// this check to always return true.
+    /// </summary>
+    public bool isJoystickDetected {
+      get { return string.IsNullOrEmpty(deviceJoystickTokens) || _isJoystickDetected; }
+    }
+
+    private float _pollTimer = 0f;
+
+    private void fixedUpdatePollConnection() {
+      if (_pollConnection && !_isJoystickDetected) {
+        _pollTimer += Time.fixedDeltaTime;
+      }
+
+      if (_pollTimer >= pollConnectionInterval) {
+        _pollTimer = 0f;
```

Comment:
@Amarcolina taught me this one: rather than setting the timer to zero, you want to subtract `pollConnectionInterval` from the current timer value.

This keeps you from losing the bits of time that `Time.fixedDeltaTime` causes `_pollTimer` to _exceed_ `pollConnectionInterval`.